playerSDK/src/main/java/com/kaltura/playersdk/KPPlayerConfig.java

```diff
@@ -274,4 +280,8 @@ public void setAdPreferedBitrate(int adPreferedBitrate) {
 	public int getAdPreferedBitrate() {
 		return mAdPreferedBitrate;
 	}
+
+	public int getPrefferedBR() {
```

Comment:
Please rename to `getPreferredBitrate()`
Also rename `prefered` to `preferred` everywhere (https://en.wiktionary.org/wiki/prefered).